css/styles.css

```diff
@@ -709,3 +710,458 @@ a {
+    flex-direction: column;
+    align-items: flex-start;
+    width: 496px;
+    height: 76px;
+    color: #344563;
+    padding: 0;
+    gap: 12px;
+    flex: 0 0 auto;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .primary-buttons {
+    display: flex;
+    margin-left: 26px;
+    list-style-type: none;
+    padding: 0;
+    margin-top: 2px;
+    gap: 8px;
+    width: 203px;
+    height: 24px;
+  }
+
+  .tonic-title h2 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    margin-left: 26px;
+    gap: 12px;
+    width: 295px;
+    height: 36px;
+    flex: 0 0 auto;
+    line-height: 44px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    color: #172b4d;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .see-project {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 17px;
+    line-height: 24px;
+    box-sizing: border-box;
+    display: flex;
+    flex: 0 0 auto;
+    flex-direction: row;
+    align-items: center;
+    text-align: center;
+    color: #396df2;
+    gap: 10px;
+    padding: 12px;
+    width: 124px;
+    height: 48px;
+    margin-left: 26px;
+    left: 0;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    background-color: #fff;
+  }
+
+  .multi-post h2 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-left: 26px;
+    margin-top: -12px;
+    padding: 0;
+    width: 250px;
+    height: 80px;
+    flex: 0 0 auto;
+    line-height: 44px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    color: #172b4d;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .about-me-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 16px;
+    margin-top: 50px;
+    margin-left: 5px;
+    margin-right: 5px;
+    margin-bottom: 20px;
+    gap: 0;
+    width: 100%;
+    flex: 0 0 auto;
+    order: 0;
+    flex-grow: 0;
+    background-color: #fff;
+    border-bottom-right-radius: 80px;
+  }
+
+  .about-myself h2 {
+    display: flex;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 700;
+    font-size: 50px;
+    font-style: normal;
+    line-height: 52px;
+    width: 32px;
+    height: 101px;
+    align-items: flex-start;
+    color: #172b4d;
+    padding: 16px;
+  }
+
+  .text-paragraph section p {
+    margin-top: 0;
+    font-style: normal;
+    font-size: 17px;
+    line-height: 24px;
+    width: 327px;
+    font-family: 'Poppins', sans-serif;
+    text-align: justify;
+    color: #344563;
+  }
+
+  .aboutme-container {
+    display: flex;
+    width: 100%;
+  }
+
+  .person-details {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 50%;
+  }
+
+  .language-section {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 50%;
+  }
+
+  .get-resume {

```

Comment:
- [ ] Let's fix the orientation of "Get My Resume" text inside the button . Text should appear in the middle. Cheers 👍  


Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/179270496-4f4b8a12-2f2a-405e-93c8-6af49946bb75.png) |  ![image](https://user-images.githubusercontent.com/60618877/179270215-67c0512f-a461-4b0d-b01c-8957bd388269.png)